pkgs/development/tools/faas-cli/default.nix

```
@@ -10,14 +10,14 @@ in
 buildGoModule rec {
   pname = "faas-cli";
   # When updating version change rev.
-  version = "0.14.2";
+  version = "0.14.3";
   rev = "b1c09c0243f69990b6c81a17d7337f0fd23e7542";
```

Could you also update `rev` as stated in code comment.
```suggestion
  rev = "56b1a7db771a4f2062ec8d57a782d27a039ba9eb";
```

---

or we just drop that part. It is informational anyway.

---

Yes, you are right. This is inconvenient. Let's do the same thing we did with arkade and k3sup en just hardcode the rev to `ref/tags/${version}`

---

thanks for suggestion :) I've just updated it